File: level76/classwork/codwars2.js
//https://www.codewars.com/kata/5a63948acadebff56f000018/train/javascript
function maxProduct(numbers, size){
  return numbers
  .sort((a, b) => b - a)
  .slice(0, size)
  .reduce((sum, el) => sum * el, 1);
}

File: level84/classwork/codwars2.py
#https://www.codewars.com/kata/5626b561280a42ecc50000d1/train/python
def dig_pow(n):
    return sum(int(x)**y for y,x in enumerate(str(n), 1))

def sum_dig_pow(a, b): 
    return [x for x in range(a,b + 1) if x == dig_pow(x)]